.github/release-drafter.yml

```diff
@@ -0,0 +1,26 @@
+categories:
+  - title: 'Major Changes'
+    labels:
+      - 'major'  # c6476b
+  - title: 'Minor Changes'
+    labels:
+      - 'feature'  # 006b75
+      - 'enhancement'  # ededed
+      - 'performance'  # 555555
+  - title: 'Bugfixes'
+    labels:
+      - 'fix'
+      - 'bugfix'
```

Comment:
Do we need to add the `fix` and `bugfix` labels? These don't exist in the pr labels list. Is this required by release drafter?

Reply:
No need, also the bot will not complain or create missing labels.